blocks/carousel/carousel.css

```diff
@@ -192,52 +214,141 @@ main .section div.carousel-wrapper {
+    font-family: var(--serif-font);
+    font-weight: var(--font-weight-light);
+    line-height: var(--line-height-120);
+    letter-spacing: 0.035rem;
+    margin: var(--spacer-element-07) 0 0 0;
 }
-
-.carousel ul.carousel-dots {
-    margin-top: var(--spacer-layout-02);
+  
+.carousel.case-study .carousel-slide-container .carousel-slide .case-study__container-wrapper .case-study__content-wrapper > p:nth-child(2) {
+    font-size: var(--font-size-16);
+    font-family: var(--sans-serif-font-light);
+    font-weight: var(--font-weight-light);
+    line-height: var(--line-height-160);
+    letter-spacing: 0.01rem;
+    margin: var(--spacer-element-08) 0 0 0;
+}
+  
+.carousel.case-study .carousel-slide-container .carousel-slide .case-study__container-wrapper .case-study__content-wrapper .button-container {
+    margin: var(--spacer-element-08) 0 0 0;
+}
+  
+.carousel.case-study .carousel-slide-container .carousel-slide .case-study__container-wrapper .case-study__content-wrapper .button-container a {
+    color: var(--neutral-white);
+    border: 2px solid var(--neutral-white);
+    background-image: none;
+    white-space: nowrap;
 }
 
-.carousel .carousel-slide {
-    width: 100%;
-    height: 100%;
+.carousel.case-study .carousel-slide-container .carousel-slide .case-study__container-wrapper .case-study__content-wrapper .button-container a:hover {
+    background-image: var(--gradient-to-right);
+    transition: all .4s ease-in-out;
+}
+  
+.carousel.case-study .carousel-slide-container .carousel-slide > div.only-picture > picture {
+    display: flex;
     position: absolute;
-    transition: all 0.5s;
+    bottom: 0;
 }
 
-.carousel .carousel-slide div {
-    display: flex;
-    flex-direction: column;
-    font-family: var(--serif-font);
-    font-weight: var(--font-weight-light);
-    line-height: 124%;
-    font-size: 31px;
+/* Media queries */
+
+/* Tablet */
+@media screen and (min-width: 768px) {
+    .carousel.case-study ul.carousel-dots {
+        margin-top: var(--spacer-layout-04);
+        gap: 10px;
+    }
+  
+    .carousel.case-study .carousel-slide-container {
+        min-height: 588px;
+    }
+  
+    .carousel.case-study .carousel-slide-container .carousel-slide .case-study__container-wrapper {
+        padding: var(--spacer-layout-06) var(--spacer-layout-05) var(--spacer-layout-04) var(--spacer-layout-05);
+    }
+
 }
 
+/* Desktop */
+@media screen and (min-width: 1200px) {
+    .carousel-container {
+        background: var(--neutral-bone);
+        padding: var(--spacer-layout-06) 0 var(--spacer-layout-06);
+    }
+
+    main .section div.carousel-wrapper {
+        padding-top: var(--spacer-layout-07);
+        padding-bottom: var(--spacer-layout-04);
+    }
+
+    .section.carousel-container .default-content-wrapper {
+        padding-top: 0;
+        padding-bottom: var(--spacer-layout-07);
+    }
+
+    .carousel .carousel-slide-container {
+        width: 100%;
+        max-width: 740px;
```

Comment:
```suggestion

```

No need to set this max-width value as we already have a max-width for the inner section div.